packages/frontend/src/ui/select/styles.module.css

```diff
@@ -36,5 +36,5 @@
 }
 
 .pickedOption {
-    @apply bg-gray-300 hover:bg-gray-300;
+    @apply bg-gray-100 hover:bg-gray-300;
```

Comment:
I think we should keep a darker color here, otherwise the selected item has the same color as the background.

Reply:
okay, I was just trying to copy Figma colors.